fl4health/parameter_exchange/parameter_selection_criteria.py

```diff
@@ -178,41 +177,72 @@ def smallest_increase_in_magnitude_scores(model: nn.Module, initial_model: Optio
     return names_to_scores
 
 
+# Helper functions for select_scores_and_sample_masks
+def _sample_masks(score_tensor: Tensor) -> NDArray:
+    bernoulli_probabilities = torch.sigmoid(score_tensor).cpu().numpy()
+    # Perform Bernoulli sampling.
+    binary_mask = bernoulli.rvs(bernoulli_probabilities)
+    return binary_mask
+
+
+def _process_masked_module(
+    module: nn.Module, model_state_dict: Dict[str, Tensor], module_name: Optional[str] = None
+) -> Tuple[NDArrays, List[str]]:
+    """
+    Perform Bernoulli sampling using the weight and bias scores of a masked module.
+
+    Args:
+        module (nn.Module): the module upon which operations described above are performed.
+            "module" can either be a submodule of the model trained in FedPM, or it can be the model itself.
```

Comment:
I might suggest a small change to this comment. Rather than saying "it can be the model itself" I would suggest, "it can a standalone module itself" I got a little confused and thought that you intended this to work for a full model being passed in as `module`, and couldn't figure out how that would work. However, in visiting the other code, I saw what you meant.